lxd/api_cluster.go

```diff
@@ -3503,6 +3514,13 @@ func restoreClusterMember(d *Daemon, r *http.Request) response.Response {
 			return err
 		}
 
+		// If "status-only" mode, update the database state without affecting instances.
+		if statusOnly {
```

Comment:
lets flip this logic so that the instance restore logic is wrapped in an if statement for skipInstances.